enterprise/frontend/src/embedding-sdk/components/private/SdkUsageProblem/SdkUsageProblemDisplay.unit.spec.tsx

```diff
@@ -0,0 +1,158 @@
+import userEvent from "@testing-library/user-event";
+
+import { setupEnterprisePlugins } from "__support__/enterprise";
+import {
+  setupPropertiesEndpoints,
+  setupSettingsEndpoints,
+} from "__support__/server-mocks";
+import { mockSettings } from "__support__/settings";
+import { renderWithProviders, screen, within } from "__support__/ui";
+import * as IsLocalhostModule from "embedding-sdk/lib/is-localhost";
+import {
+  createMockApiKeyConfig,
+  createMockJwtConfig,
+} from "embedding-sdk/test/mocks/config";
+import { createMockSdkState } from "embedding-sdk/test/mocks/state";
+import type { SDKConfig } from "embedding-sdk/types";
+import {
+  createMockSettings,
+  createMockTokenFeatures,
+  createMockUser,
+} from "metabase-types/api/mocks";
+import { createMockState } from "metabase-types/store/mocks";
+
+const TEST_USER = createMockUser();
+
+jest.mock("metabase/visualizations/register", () => jest.fn(() => {}));
+
+interface Options {
+  config: SDKConfig;
+  hasEmbeddingFeature?: boolean;
+}
+
+const setup = (options: Options) => {
+  const tokenFeatures = createMockTokenFeatures({
+    embedding_sdk: options.hasEmbeddingFeature ?? true,
+  });
+
+  const settingValues = createMockSettings({ "token-features": tokenFeatures });
+
+  const state = createMockState({
+    settings: mockSettings(settingValues),
+    currentUser: TEST_USER,
+    sdk: createMockSdkState(),
+  });
+
+  setupEnterprisePlugins();
```

Comment:
Should we have this, as we don't have the plugins in the actual sdk?

Reply:
@npretto good point, I'll get rid of that.

Reply:
Fixed in https://github.com/metabase/metabase/pull/47821/commits/7465abcb6b00e1f448e19d581c1a6022258b777c, thanks!